lib/teiserver/mix_tasks/fake_data.ex

```diff
@@ -458,4 +459,24 @@ defmodule Mix.Tasks.Teiserver.Fakedata do
       :rand.uniform() < chance
     end)
   end
+
+  # Adds a party id or nil
+  defp get_party_id(num_players) do
+    case is_in_party?() do
+      true ->
+        num_parties = trunc(num_players / 4)
+        # party id is a string
+        "#{Enum.random(1..num_parties)}"
+
+      false ->
+        nil
+    end
```

Comment:
So... an if clause? 😁 Ah, but you want to output that string or nil... I guess, yeah, it works :)